src/plugin/pid/pidwrappers.h

```diff
@@ -130,7 +135,9 @@ extern "C"
   MACRO(__xstat64)          \
   MACRO(__lxstat)           \
   MACRO(__lxstat64)         \
-  MACRO(readlink)
+  MACRO(readlink)           \
+  MACRO(process_vm_readv)   \
```

Comment:
Can we put these MACROs in their own category and include them in PIDVIRT_GEN_ENUM only if the kernel version matches?